app/BUILD.bazel

```
@@ -281,8 +281,11 @@ Source files for the views library that DO NOT import resources.
 Place your file here if:
  - It is a view file
  - It does not import resources
+ - It holds a circular dependency with a view (including views in VIEWS_WITH_RESOURCE_IMPORTS)
 '''
 VIEWS = [
+    "src/main/java/org/oppia/app/application/ApplicationInjectorProvider.kt",
+    "src/main/java/org/oppia/app/application/ApplicationInjector.kt",
```

```suggestion
    "src/main/java/org/oppia/app/application/ApplicationInjector.kt",
    "src/main/java/org/oppia/app/application/ApplicationInjectorProvider.kt",
```

Swapped for correct sorting.